### PR TITLE
rebrand bluemix to ibm cloud

### PR DESCRIPTION
**Pull request by ishangulhane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,11 +36,11 @@
 2. Under project setting > info > Url Types, Add $(PRODUCT_BUNDLE_IDENTIFIER) as a URL Scheme
 3. Add the following import to your AppDelegate.swift file:
 	```swift
-	import BluemixAppID
+	import
```
Comment (sandmman) [inline README.md]: Seems you lost your import name

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@
 After the App ID client SDK is initialized, you can start authenticating users by launching the Login Widget.
 1. Add the following import to the file in which you want to use with the login Widget:
 ```swift
-import BluemixAppID
+import
```
Comment (sandmman) [inline README.md]: here too

File: README.md [hunk 3]
```diff
@@ -265,11 +265,11 @@ userProfileManager?.getUserInfo(accessTokenString: accessToken, identityTokenStr
 Add the following imports to the file in which you want to invoke a protected resource request:
 ```swift
 import BMSCore
-import BluemixAppID
+import
```
Comment (sandmman) [inline README.md]: and here

File: README.md [hunk 4]
```diff
@@ -280,8 +280,8 @@ request.send(completionHandler: {(response:Response?, error:Error?) in
 ## License
 This package contains code licensed under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 and may also view the License in the LICENSE file within this package.
 
-[img-bluemix-powered]: https://img.shields.io/badge/bluemix-powered-blue.svg
-[url-bluemix]: http://bluemix.net
+[img-ibmcloud-powered]: https://img.shields.io/badge/ibmcloud-powered-blue.svg
```
Comment (sandmman) [inline README.md]: I think I prefer the capitalized, spaced version `IBM%20Cloud`. Are there similar markdown files that display it your way?

File: Source/BluemixAppID/api/AppID.swift [hunk 5]
```diff
@@ -17,32 +17,32 @@ import BMSCore
...
-    
+
     internal init() {}
-    
+
     /**
         Intializes the App ID instance
         @param tenantId The tenant Id.
-        @param bluemixRegion The bluemix region.
+        @param region The IBm Cloud region.
```
Comment (sandmman) [inline Source/BluemixAppID/api/AppID.swift]: Might as well fix the capitalization here.

File: README.md [hunk 6]
```diff
@@ -18,12 +18,12 @@
```
Comment (sandmman) [inline README.md]: We require iOS 10 now

File: README.md (hunk 4 above)
Comment (ishangulhane) [reply README.md]: To keep it consistent with Node SDK, I am displaying `ibmcloud powered`

Comment (sandmman) [reply README.md]: screen shot 2018-06-05 at 11 13 50 am

Where do you mean? This is master/development as I see it

Comment (ishangulhane) [reply README.md]: We are changing the NPM module name to `ibmcloud-appid`. To reflect the change, I am updating the icon to `ibmcloud powered`. Also starter repo has a single icon for all the sample apps
screen shot 2018-06-05 at 11 24 58 am

To maintain consistency, I kept it as `ibmcloud`. If required we can update it.

Comment (sandmman) [reply README.md]: I don't really see the need for those to match. The readmes should reflect our corporate branding, which is `IBM Cloud`. In the past, we have join words others words like `appid` in repo/field names likely to limit the number of dashes, but whenever there is an actually branding situation the proper format is displayed (as in the line above the powered by bluemix in your screenshot).

**Pull request merged**